install-and-configure/install/multi-cluster/federated-etl/aggregator.md

```diff
@@ -77,74 +69,97 @@ Because the Aggregator PV is relatively small, the least expensive performance g
 
 ```yaml
 kubecostAggregator:
-  env:
-    # This interval defines how long the Aggregator spends ingesting ETL data
-    # from the federated store bucket into SQL tables, before exiting its job to
-    # enter the derivation step. If set too low for large scale users, the
-    # Aggregator may not have enough time to ingest all new data that exists in
-    # the federated store bucket. If set too high, there will be a delay in data
-    # between the Kubecost Agents and the Aggregator.
-    #
-    # Note, that the default value is set to 60m to optimize for the
-    # first-install experience of Kubecost (i.e. it prioritizes small data
-    # becoming available more quickly).
-    #
-    # default: 60m
-    DB_BUCKET_REFRESH_INTERVAL: 2h
+  # deployMethod determines how Aggregator is deployed. Current options are
+  # "singlepod" (within cost-analyzer Pod) "statefulset" (separate
+  # StatefulSet), and "disabled". Only use "disabled" if this is a secondary
+  # Federated ETL cluster which does not need to answer queries.
+  deployMethod: singlepod
+
+  # fullImageName overrides the default image construction logic. The exact
+  # image provided (registry, image, tag) will be used for aggregator.
+  # fullImageName:
+  imagePullPolicy: IfNotPresent
+
+  # For legacy configuration support, `enabled: true` overrides deployMethod
+  # and causes `deployMethod: "statefulset"`
+  enabled: false
+
+  # Replicas sets the number of Aggregator replicas. It only has an effect if
+  # `deployMethod: "statefulset"`
+  replicas: 1
```

Comment:
@jessegoodier In fact we should probably remove this whole section because these are already the defaults in the `values.yaml` file?